### PR TITLE
Moved some code for prompting a user to utils so it can be used by ot…

### PR DESCRIPTION
**Pull request by lobabob**
…her commands.

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -22,7 +26,38 @@ function parseRoleId(role) {
   return matches && matches[1] || -1
 }
 
+/**
+ * Responds to a message with a prompt and returns the answer.
+ *
+ * @param {Message} msg Message to respond to
+ * @param {boolean} [sameAuthor] Whether the answer should be from the same author as the initial message
```
Comment (keitharm) [inline lib/utils.js:33]: Add default value for `sameAuthor` in the doc

File: lib/utils.js [hunk 2]
```diff
@@ -22,7 +26,38 @@ function parseRoleId(role) {
...
+ * being prompting from
+ * @returns {Promise}
+ */
+async function prompt(msg, sameAuthor = true) {
+  const collector = new MessageCollector(msg.channel, m => {
```
Comment (keitharm) [inline lib/utils.js:38]: Consider using `msg.channel.createMessageCollector` and also removing the corresponding MessageCollector import.

Comment (lobabob) [reply lib/utils.js]: done

File: lib/utils.js (hunk 1 above)
Comment (lobabob) [reply lib/utils.js]: done

**Pull request merged**